signup.py

```diff
@@ -74,3 +75,64 @@ def signup_via_webform(user_data, activate_right_away=True):
             click_button(driver, class_name = "skip") #skips verification step
 
         _signup_step_betaling(driver)
+
+def signup_via_medlemshjaelp(vagt_user_data, signup_data, payment="mobilepay"):
+    """Log in as a Butiksvagt and creates a user with the requested email address using the medlemshjælp page.
+
+    Positional arguments:
+        signup_data -- a dictionary containing the following keys: 
+            "firstname", 
+            "lastname",
+            "email",
+            "membershiptype" (in ["Frivillig", "Støttemedlem"])
+            "department" (in ["Vesterbro", "Amager"])
+        vagt_user_data -- a dictionary containing the following keys:
+            "email",
+            "password" -- used to log in as existing member on butiksvagt
+        payment (in ["mobilepay", "cash", "skip"])
+
+    This method uses its own selenium driver, so doesn't disturb the flow of the enclosing test"""
+    with webdriver.Firefox() as driver:
+        try_login(driver, vagt_user_data["email"], vagt_user_data["password"])
+        assert_current_page_is("min_side", driver)
+        navigate_to_page("medlemshjaelp-signup", driver)
+        assert_current_page_is("medlemshjaelp-signup", driver)
+        
+        fill_form_field(signup_data["firstname"], driver, form_id="input_firstname")
+        fill_form_field(signup_data["lastname"], driver, form_id="input_lastname")
+        fill_form_field(signup_data["email"], driver, form_id="input_email")
+        fill_form_field(signup_data["email"], driver, form_id="input_confirm_email")
+        select_from_dropdown(signup_data["membershiptype"], driver, "input_item_id")
+        select_from_dropdown(signup_data["department"], driver, "input_department_id")
+        check_checkbox(driver, "input_terms")
+        submit_form(driver)
+
+        wait_for_next_page(driver)
+        
+        if payment == "mobilepay":
+            check_checkbox(driver, "input_confirm_mobilepay_payment")
+            
+            # mobilepay payment submit button
+            click_button(driver, xpath="/html/body/div/div[3]/div/div/form[1]/ul/li/input")
```

Comment:
There are almost certainly better ways to do this - e.g. the form has a class "mobilepay" so you should be able to use an xpath along the lines of `//form[@class="mobilepay"]/ul/li/input` (this may not be the precise xpath format, but there's some similar functionality)

Reply:
Right, this was before I understood that xpaths can do that :)